pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/jstl/JstlFunctions.java

```diff
@@ -53,6 +53,12 @@ public static Object coalesce(Object value, Object valueIfNull) {
     return value == null ? valueIfNull : value;
   }
 
+  public static String replace(Object input, Object regex, Object replacement) {
```

Comment:
Existing methods doesn't fail if any arg is null. Would be sense here to return `input` if either `regex` or `replacement` is null?

Reply:
done